Program.cs

```diff
@@ -0,0 +1,405 @@
+
+            Console.WriteLine("Id Seller: ");
+            if (!int.TryParse(Console.ReadLine(), out int idSeller))
+            {
+                return;
+            }
+
+            Sellers objSellers = sellersList.SingleOrDefault(vendedor => vendedor.IdSeller == idSeller);
+            PrintData(objSellers);
+
+            //get the last ID
+            int maxIdInvoice = 1;
+            if (invoicesList.Count > 0) //Another method
+            {
+                maxIdInvoice = invoicesList.Max(c => c.IdInvoice) + 1;
+            }
+
+            //The header is recorded
+            Invoices invoices = new Invoices();
+            invoices.IdInvoice = maxIdInvoice;
+            invoices.Customers = objCustomers;
+            invoices.Sellers = objSellers;
+
+            string optionSelected = "y";
+            int numberRegistry = 0;
+            decimal total = 0;
+
+            invoices.InvoiceDetails = new List<InvoiceDetails>();
+
+            while (optionSelected == "y")
+            {
+                Console.WriteLine("Id Product: ");
+                if (!int.TryParse(Console.ReadLine(), out int idProduct))
+                {
+                    return;
+                }
+
+                Products objProducts = productsList.SingleOrDefault(producto => producto.IdProduct == idProduct);
+                PrintData(objProducts);
+
+                Console.WriteLine("Quantity: ");
+                if (!int.TryParse(Console.ReadLine(), out int quantity))
+                {
+                    return;
+                }
+
+                InvoiceDetails invoiceDetails = new InvoiceDetails();
+                invoiceDetails.IdInvoiceDetail = numberRegistry + 1;
+                invoiceDetails.Products = objProducts;
+                invoiceDetails.Price = objProducts.Price;
+                invoiceDetails.Quantity = quantity;
+
+                total = total + objProducts.Price*quantity;
+
+                invoices.InvoiceDetails.Add(invoiceDetails);
+
+                numberRegistry++;
+
+                optionSelected = ConfirmationAddDetails();
+            }
+
+            invoices.Amount = total;
+            invoicesList.Add(invoices);
+
+            MenuMain();
+            ProcessMain();
+        }
+        private static void ReportSales()
+        {
+            decimal TotalAmount = 0;
+
+            Console.WriteLine("Sales Record");
+
+            invoicesList.ForEach(itemInvoices => {
+                TotalAmount = TotalAmount + itemInvoices.Amount;
+                itemInvoices.InvoiceDetails.ForEach(itemInvoiceDetails =>
+                {
+                        Console.WriteLine($"Id Invoice: {itemInvoices.IdInvoice}  Id Invoice Detalle: {itemInvoiceDetails.IdInvoiceDetail} Product: {itemInvoiceDetails.Products.Description}   Customer: {itemInvoices.Customers.Name} Price: {itemInvoiceDetails.Price}    Quantity: {itemInvoiceDetails.Quantity} Total: {itemInvoiceDetails.Price * itemInvoiceDetails.Quantity}");

```

Comment:
the alignment must to be by 4 characters or a single tab, You must to have clear, that our client always going to see our code so we have to write a beautiful code and be careful with standards and bugs.